services/camara/partido.js

```diff
@@ -0,0 +1,55 @@
+import fetch from 'node-fetch';
+import PartidosError from '../../errors/PartidosError';
+
+const endPoint = 'https://dadosabertos.camara.leg.br/api/v2/partidos';
+
+const requestOptions = {
+  headers: {
+    'Accept': 'application/json'
+  }
+};
+
+async function getPartido(data) {
```

Comment:
Acho que o que vou comentar agora pode ser um pouco de gosto, mas como não utilizamos tipagem e nem nada do gênero, uma coisa que acho interessante é receber os parâmetros em forma de dicionário, assim fica muito mais claro para quem for utilizar quais dados devem ser passador
```suggestion
async function getPartido({ partido, incluirMembros }) {
```